pkg/services/apiserver/builder/runner/builder.go

```diff
@@ -119,6 +120,11 @@ func (b *appBuilder) UpdateAPIGroupInfo(apiGroupInfo *genericapiserver.APIGroupI
 				return err
 			}
 			apiGroupInfo.VersionedResourcesStorageMap[version][resourceInfo.StoragePath()] = store
+			if registryStore, ok := store.(*genericregistry.Store); ok {
+				for subPath, _ := range kind.ZeroValue().GetSubresources() {
```

Comment:
I think this is a good initial quick fix, given that the generated code for `GetSubresources` always returns every declared subresource. The [declaration in the interface](https://github.com/grafana/grafana-app-sdk/blob/main/resource/object.go#L36) doesn't require that the the subresource be in the map if unpopulated, which means we may want to enhance the app-sdk to make this more robust in the future.